Binance Detect Moonings.py

```diff
@@ -129,8 +129,10 @@ def wait_for_price():
 
     if historical_prices[hsp_head]['BNB' + PAIR_WITH]['time'] > datetime.now() - timedelta(minutes=float(TIME_DIFFERENCE / RECHECK_INTERVAL)):
         # sleep for exactly the amount of time required
-        time.sleep((timedelta(minutes=float(TIME_DIFFERENCE / RECHECK_INTERVAL)) - (datetime.now() - historical_prices[hsp_head]['BNB' + PAIR_WITH]['time'])).total_seconds())
-
+        sleep_time = (timedelta(minutes=float(TIME_DIFFERENCE / RECHECK_INTERVAL)) - (datetime.now() - historical_prices[hsp_head]['BNB' + PAIR_WITH]['time'])).total_seconds()
+        time.sleep(sleep_time)
+        print(f'Not enough time has passed yet, will wait {sleep_time} {"seconds" if sleep_time > 1 else "second"}...')
+        
     print(f'not enough time has passed yet...Session profit:{session_profit:.2f}%')
```

Comment:
If we are tidying up the messaging shouldn't this be added into line 134? Otherwise I believe it will print 2 lines.
I.e. every X seconds it will print:
`Not enough time has passed yet, will wait 3 seconds...`
`not enough time has passed yet...Session profit:1.00%`

Changing line 134 to something like: `print(f'Not enough time has passed yet, will wait {sleep_time} {"seconds" if sleep_time > 1 else "second"}... Session profit: {session_profit:.2f}%')` would bring it all nicely to a single line.